app/Http/Requests/Three/SignupUpdateRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
is this weird to have separate requests for the update and the creation of a signup? Should this validation just live in the controller?

Reply:
I'm personally a fan, since it's a little easier to reason about (especially if you want to drop a `dd()` before the validation). And it's also nice if you want to share a base set of rules between controller methods. 👍

Reply:
awesome, updated @DFurnes !